File: func_interchange.cpp
#include <iostream>
using namespace std;
void interchange(int a,int b,int c){
    int temp;
    temp = a;
        a = b;
        b = c;
        c = temp;


cout<<"value of a is = "<<a<<endl;
cout<<"value of b is = "<<b<<endl;
 cout<<"value of c is = "<<c<<endl;   
}
int main() {
    
    int a,b,c;

cout<<"enter the value of a = ";
cin>>a;
cout<<"enter the value of b = ";
cin>>b;
cout<<"enter the value of c = ";
cin>>c;

interchange(a,b,c);
    return 0;
}

File: func_interchange.java
import java.util.Scanner;

public class Interchange {
    public static void interchange(int a, int b, int c) {
        int temp = a;
        a = b;
        b = c;
        c = temp;

        System.out.println("Value of a is = " + a);
        System.out.println("Value of b is = " + b);
        System.out.println("Value of c is = " + c);
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter the value of a: ");
        int a = scanner.nextInt();

        System.out.print("Enter the value of b: ");
        int b = scanner.nextInt();

        System.out.print("Enter the value of c: ");
        int c = scanner.nextInt();

        interchange(a, b, c);
        
        scanner.close();
    }
}
